### PR TITLE
Identifies connections to Solorigate-related DNS records

### PR DESCRIPTION
**Pull request by yoweiz**
Identifies connections to Solorigate-related DNS records based on VM insights data.
Based on: https://github.com/Azure/Azure-Sentinel/blob/master/Detections/MultipleDataSources/Solorigate-Network-Beacon.yaml

### CODE REVIEW

File: Detections/MultipleDataSources/Solorigate-VM-Network.yaml
Comment (YaronFruchtmann) [inline Detections/MultipleDataSources/Solorigate-VM-Network.yaml:29]: Should be 1h

**Review by YaronFruchtmann — APPROVED** [review]
This is fine. connection table is expected to be small, so it is, as it should be, the first in join operator. I am approving but @shainw knows Solarigate better, and I prefer  he has a look before merge

**Review by laithhisham — COMMENTED** [review]
Looks good :)
I also recommend waiting for Shain's approval as Yaron said

**Review by shainw — CHANGES_REQUESTED** [review]
I don't see why we would need a separate query on this, I think we should just update this - https://github.com/Azure/Azure-Sentinel/blob/master/Detections/MultipleDataSources/Solorigate-Network-Beacon.yaml

**Comment by yoweiz** [comment]
> I don't see why we would need a separate query on this, I think we should just update this - https://github.com/Azure/Azure-Sentinel/blob/master/Detections/MultipleDataSources/Solorigate-Network-Beacon.yaml

I understood from @tamersalman that you both agreed to write a separated detection. Please correct me if I'm wrong. We currently have a limitation on the number of entities, so we won't be able to add file and process entities to the previous detection.

**Comment by shainw** [comment]
> > I don't see why we would need a separate query on this, I think we should just update this - https://github.com/Azure/Azure-Sentinel/blob/master/Detections/MultipleDataSources/Solorigate-Network-Beacon.yaml
> 
> I understood from @tamersalman that you both agreed to write a separated detection. Please correct me if I'm wrong. We currently have a limitation on the number of entities, so we won't be able to add file and process entities to the previous detection.

Got it, lets make sure the name is clear then.  Commented PR again for name.

File: Detections/MultipleDataSources/Solorigate-VM-Network.yaml
Comment (shainw) [inline Detections/MultipleDataSources/Solorigate-VM-Network.yaml:2]: Since we will have 2 of these very similar but against different data sets we should be clear in the name.  I suggest something like - "Solorigate Domains Found in VM Insights"

**Review by shainw — CHANGES_REQUESTED** [review]

**Review by shainw — APPROVED** [review]

**Pull request merged**
